st2common/st2common/util/jsonify.py

```diff
@@ -67,8 +70,8 @@ def json_loads(obj, keys=None):
     for key in keys:
         try:
             obj[key] = json.loads(obj[key])
-        except:
-            pass
+        except Exception:
+            LOG.exception("Criteria pattern not valid JSON.")
```

Comment:
I noticed this issue while testing latest dev build - this exception is not fatal and we shouldn't log anything (or use debug log level) since it will cause unnecessary noise, churn and confusion in the action runner.

The method is named ``try_loads`` and it uses best-effort approach where dictionary item not being JSON  does not represent an error.

I will make that change in another PR - remove log statement and add a comment on why we don't log anything (error is not fatal).

/cc @m4dcoder

Reply:
b2ed03b54